suite-native/atoms/src/Modal/BottomModal.tsx

```diff
@@ -56,32 +127,47 @@ export const BottomModal = ({
 }: BottomModalProps) => {
     const { applyStyle } = useNativeStyles();
     const insets = useSafeAreaInsets();
+    const { animatedModalWrapperStyle, closeModalAnimated, resetModalAnimated, panGestureEvent } =
+        useAnimationStyles({ onVisibilityChange });
```

Comment:
@Nodonisko, @juriczech not sure about the hook name `useAnimationStyles` because there are more than styles now... in the other hand this had been our convention till now... do you have some good name for this? `useAnimationProps` maybe?

Reply:
Why not just use `useBottomModalAnimation`?

Reply:
https://github.com/trezor/trezor-suite/pull/5715/commits/4c38009c30a9afa0ff6ce6598f06977c91f450a1